### ISSUE TITLE
Pretendo bot isn't adding me back.

### ISSUE THREAD

**Issue by MeleeGamer** [MCMi460/3DS-RPC#42]
I've confirmed that both codes are correct, but it's still listing it as a provisionally registered friend.

**Comment by MeleeGamer** [comment]
Nevermind, it just took a lot longer than expected.

**Comment by Preloading** [comment]
> Nevermind, it just took a lot longer than expected.

This was because we restarted the server, so it's still a bug. If you could, could you reopen it?

**Comment by Natino64** [comment]
I’m having the same issue - added a few hours ago and still no accepted friend. 
Not sure if that’s expected performance, just chiming in.

**Comment by SepticRedhead** [comment]
I'm also having this issue. I guess it's still a work in progress or too many people are registering at once.

**Comment by Ams095** [comment]
Same for me

**Comment by Getze56901** [comment]
> Same for me

Samee, have you fixed yet? mine's appears in "NONE" and Status 404
![image](https://github.com/user-attachments/assets/cebc0295-37c3-497b-838f-0cfeee933272)
![image](https://github.com/user-attachments/assets/2dd500b8-f053-407e-9af6-1a675facaeb5)

**Comment by usersoni2** [comment]
also me

**Comment by chimmmmy** [comment]
i also still have this issue

**Comment by chimmmmy** [comment]
oh it friended me back

but game status didn't update, its still stuck on mii plaza for some reason

**Comment by chasio2221** [comment]
the website has hard times updating game statuses after 3 hours.

**Comment by bvnovaes** [comment]
Today I try to add Pretendo as a friend and apparently the bug still persists. Any news about that?

**Comment by alex-628** [comment]
From what I can tell, the service is just slow as hell. _**Not broken,**_ but just slow. 💔

**Comment by funkycharlie** [comment]
Happening to me too, hope it friends me back eventually.

**Comment by walrein** [comment]
still occurring

**Comment by cobaltw** [comment]
Also having this issue

**Comment by ybgmoli** [comment]
Yup having the same issue here. Nearly a full day has passed and the bot hasn't added me back.

**Comment by Lordpun** [comment]
Is this an issue with specifically Pretendo, or would connecting to Nintendo's official servers make it update the discord status faster? At least assuming that still works post E-Shop closure.

**Comment by Preloading** [comment]
> Is this an issue with specifically Pretendo, or would connecting to Nintendo's official servers make it update the discord status faster? At least assuming that still works post E-Shop closure.

This is an issue specifically with pretendo, Nintendo is much faster

**Comment by chasio2221** [comment]
More like a bot problem then a pretendo problem

**Comment by Preloading** [comment]
> More like a bot problem then a pretendo problem

I mean we do run the same code* on both servers, so take that what you will
Also, by Pretendo, we were refrencing the Pretendo bot

**Comment by Devnol** [comment]
Same issue here, registered yesterday and still says provisionally added.

**Comment by MayaChen350** [comment]
I've been waiting for finally getting a status since a while
the bot never added me and I tried adding it back but still don't have it friended
Oh well since I'm kinda looking forward for this

Is there any way I could help?

Thanks

**Comment by Devnol** [comment]
I'm starting to think the bot registers users only when the pretendo backend is restarted (currently 10 days of uptime). Is it possible to give it a reboot and see if that does the trick?

**Comment by MayaChen350** [comment]
Might I also ask:

How is my "Favorite game" updating on the website if the bot doesn't have me added and it's written that I was last online on April 26th (I updated my favorite game a few days ago)

**Comment by SpaceyF** [comment]
not to necropost, but same issues here

**Comment by Carocim** [comment]
Bot isn't adding me back as well.

**Comment by FireShade3DS** [comment]
Same here, on both servers

**Comment by FireShade3DS** [comment]
> Same here, on both servers

Finally connected on Nintendo. Pretendo still hasn't connected.

**Comment by FireShade3DS** [comment]
> > Same here, on both servers
> 
> Finally connected on Nintendo. Pretendo still hasn't connected.

Pretendo is finally connected but neither servers have registered the bot in my friends list and game sync data just isn't. It's not doing what it's supposed to.

**Comment by BlendNugget** [comment]
Same for me.

**Comment by Gakefon11** [comment]
Bot isn't adding me back

**Comment by MayaChen350** [comment]
> From what I can tell, the service is just slow as hell. _**Not broken,**_ but just slow. 💔

Should be handled soon hopefully, I already changed some stuff with my latest PR

I really like the project and I'm confident in my abilities

I'll do everything I can to speed things up!

**Comment by jayjayare** [comment]
Issue is still VERY prominent.
pretendo
both codes are correct

**Comment by amethyst2667** [comment]
same problem still :(

**Comment by FriesEcho** [comment]
7 days passed and the bot still hasn't friended me
Hasn't workde once in my end

**Comment by toontownlpz** [comment]
March 17th 2026, still an issue it seems. Can't get the bot to add me back.

**Comment by thestoreisinsured** [comment]
8th of april 2026, loneliness is the new black

**Comment by toontownlpz** [comment]
April 8th 2026, wife left and took the kids. Life's getting a bit difficult. Health issues arising. 3DS-RPC bot hasn't added me back. Cat's dying. Not sure how long I can go on.

**Comment by thestoreisinsured** [comment]
May 5th 2026, Cat's dead. My friends left me. I was kicked out of my home. 3DS-RPC bot has ghosted me. I hate my life.

**Comment by OmarMahmmoud2007** [comment]
May 20th, 2026, food supplies running low, enemies closing in, this may be it, boys. Go run, I'll hold them back. Be sure to give the 3DS-RPC bot my letter.  

*charges out of cover*

**Comment by kirbsnet** [comment]
May 28th, 2026, No! Don't be a hero! We'll find a better way to make 3DS-RPC add us. You still have so much Persona still to play!

**Comment by TipsyTheCat** [comment]
June 3rd, 2026: EVERYONE get in the CAR! We’re LEAVING this town! WHAT?! This ISN’T the CAR! NOO! AH!

**Comment by kirbsnet** [comment]
*BANG BANG BANG* The court is now in session for the murder of 3DS-RPC.

**Comment by Teaeat** [comment]
> June 3rd, 2026: EVERYONE get in the CAR! We’re LEAVING this town! WHAT?! This ISN’T the CAR! NOO! AH!

Yare yare daze gigi . . .
